js/src/features/human-location/components/EmployeeListItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback } from "react";
+import { List, Button, Typography } from "antd";
+import { EmployeeShort, Location, EmployeeLocation } from "LocationModels";
+import { EmployeeLocationListItem } from "./EmployeeLocationListItem";
+
+const { Text } = Typography;
+
+interface Props {
+  employee: EmployeeShort;
+  employeeLocation?: EmployeeLocation[];
+  locations: Location[];
+  selectEmployee: (id: string) => void;
+  selectEmployeeLocation: (val: EmployeeLocation | null) => void;
+  remove: (id: string) => void;
+}
+
+export const EmployeeListItem: React.FC<Props> = ({
+  employee,
+  locations,
+  employeeLocation,
+  selectEmployee,
+  selectEmployeeLocation,
+  remove,
+  ...rest
```

Comment:
No, no really. Deleted. I put them here because I remember some components for which these arguments were important. They changed component appearance a bit.